news/2024/2024-01-07-featured-artist-track-updates-winter-2024.md

```diff
@@ -0,0 +1,254 @@
+---
+layout: post
+title: "Featured Artist Track Updates: Winter 2024"
+date: 2023-01-07 19:00:00 +0000
+---
+
+Since your New Year's resolution is to start mapping, you might as well get started with one of the **50+** songs that were just added to our Featured Artist library!
+
+![](/wiki/shared/news/banners/featured-artist.jpg)
+
+While our schedule is usually packed with weekly artist releases, we're flipping the script today. We have nearly [**400** artists in our catalogue](https://osu.ppy.sh/beatmaps/artists), so we'd like to bring them back into the spotlight by updating a few of their song listings!
+
+To be 200% clear, this article highlights every newly licensed song from older Featured Artists since [our batch of summer updates](https://osu.ppy.sh/home/news/2023-07-22-featured-artist-track-updates-summer-2023).
+
+Here's how everything works:
+
+- Use the links below to skip to a specific artist in this article
+  - [**Aiobahn**](#aiobahn)
+  - [**Celldweller**](#celldweller)
+  - [**ColBreakz**](#colbreakz)
+  - [**cygnus**](#cygnus)
+  - [**HyuN**](#hyun)
+  - [**SAMString**](#samstring)
+  - [**seatrus**](#seatrus)
+  - [**Umeboshi Chazuke**](#umeboshi-chazuke)
+- Ranked/Loved maps for new songs are listed here, along with one or two audio previews per artist. Check the artist's listing for all previews and beatmap template downloads!
+- Any new songs are marked with a **NEW** icon on the Featured Artist listing
+
+![](/wiki/shared/news/2022-01-02-featured-artist-track-updates-winter-2022/new.png)
+
+Let's get started.
+
+---
+
```

Comment:
line above heading doesn't really look that great

![image](https://github.com/ppy/osu-wiki/assets/36758269/5c6f739e-bff4-4333-8644-fb384d175c68)


```suggestion
```